.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
 build
+walletKey.pem
```

Comment:
We can leave the walletKey.pem file, just in case, but it should never land in the workspace. It should be used in the separate custom directory described in the previous comment.

As for the package-lock, my general rule is always that it should be in the repository, but the person who reviews it should never merge changes as is. Instead, I can always install updates to packages by myself. And version changes are also on my site.

So here, it would be best to remove changes to package-lock and package.json